src/components/swiper/testimonials-swiper/TestimonialsSwiper.style.ts

```diff
@@ -44,35 +48,82 @@ export const TestimonialsHeading = styled(Typography)({
 
 export const TestimonialsCard = styled(Box)({
   backgroundColor: '#fff',
-  padding: '64px 30px',
+  padding: '58px 30px',
   borderRadius: '20px',
   boxShadow: '0 4px 20px rgba(0, 0, 0, 0.1)',
   width: '100%',
   textAlign: 'left',
-  height: '400px',
+  minHeight: '426px',
   display: 'flex',
   flexDirection: 'column',
 });
 
```

Comment:
![image](https://github.com/user-attachments/assets/81d2d2fa-94bf-4061-b67d-4280eaa09b8c)
I think these cards should have a min-width defined aswell as having height 100% so that if one card is larger so the other cards will match